modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -67,13 +85,15 @@
 		"Pod Jockey",
 		"Union Bitrunner",
 		"Junior Runner",
+		"Professional Gamer",
 	)
 
 /datum/job/blueshield
 	alt_titles = list(
 		"Blueshield",
 		"Command Bodyguard",
 		"Executive Protection Agent",
+		"Command Goon",
```

Comment:
Is having the word "goon" really a good idea for all the guard roles?

Reply:
Ehh, I wasn't sold on it, I'll axe it.